lib/mogrify.ex

```diff
@@ -59,12 +59,12 @@ defmodule Mogrify do
   end
 
   defp arguments_for_saving(image, path) do
-    base_arguments = ~w(-write #{path} #{String.replace(image.path, " ", "\\ ")})
+    base_arguments = ~w(-write) ++ [path] ++ [image.path]
```

Comment:
May as well get rid of the `~w()` syntax here, and just provide a single list instead of concatenating lists.